Apps/WebClient/src/ClientApp/test/stubs/store/storeOptionsStub.ts

```diff
@@ -3,22 +3,22 @@ import { injectable } from "inversify";
 
 import { GatewayStoreOptions } from "@/store/types";
 
-import authStub from "./auth";
-import commentStub from "./comment";
-import configStub from "./config";
-import encounterStub from "./encounter";
-import errorBannerStub from "./error";
-import idleStub from "./idle";
-import immunizationStub from "./immunization";
-import laboratoryStub from "./laboratory";
-import medicationStub from "./medication";
-import navbarStub from "./navbar";
-import noteStub from "./note";
-import timelineStub from "./timeline";
-import userStub from "./user";
+import authStub from "./authStub";
```

Comment:
not sure this is necessary since it's already in the stub folder
```@test/stubs/store/auth```
vs
```@test/stubs/store/authstub```

Reply:
I think same reason why Stub/store is not stub/storeStub

Reply:
the same goes for services, controllers and other objects, but we do add the suffix to make the file unique, one example is when you have multiple tabs open in vscode, by having the authStub name you automatically know what that is, otherwise you have to look at the full path to know the difference between auth vs stubs/auth

Reply:
store was renamed to storeOptionsStub

Reply:
I think @tiagograf-AOT has convinced me.

Reply:
That's where the folder name is acting as a namespace. In c# we have explicit namespaces defined so the confusion and variable collision does not happen. We can also do it here if necessary since Typescript has the concept of namespacing. 

Following the same line, we don't call the classes TestStoreOptionsStub since it is already included in the path for inclussion.

Reply:
> I can revert the changes if @MikeLyttle also thinks this is how we should name our files but in that case, the solution would be to rename the classes and drop the "stub" word

The code styles that we practice are not defined by _majority vote_ but instead _consensus_. There is always room for improvement and we should aim for the highest quality of code. If there is a disagreement about which one is better we need to articulate why and how it can be improved as well as the downsides.

Reply:
Chatted about this online (zoom) and agreed to leave the changes as is since it aligns with SonarQube's recommendations. We also agreed to work on communicating better the intent and the questions on PRs